scripts/lint.sh

```diff
@@ -112,13 +118,23 @@ if [ "$run_fast_linters" -eq 1 ]; then
     echo -e "\n*** End of copyright line check run; exit: $copyright_line_check_exit_code ***\n"
 
     echo -e '*** Running isort... ***\n'
-    isort --check-only --settings-path=setup.cfg --diff --recursive $lintable_locations
```

Comment:
the --recursive flag is enabled by default since isort 5.0.0, removing it quiets a deprecation warning.